### PR TITLE
Documentation fix

### PR DESCRIPTION
**Pull request by vamsi12a2**
Issue Number: N/A

## Summary
Documentation Fix.

## Changes

- List of comprehensive changes
1. Move 'Carbon Aware Rest API' heading to the start of the document.
2. Add missing GET endpoints and error handling links to 'Table Of Contents'.

## Checklist

- [ ] Local Tests Passing?
- [ ] CICD and Pipeline Tests Passing?
- [ ] Added any new Tests?
- [x] Documentation Updates Made?
- [ ] Are there any API Changes? If yes, please describe below.
- [x] This is not a breaking change. If it is, please describe it below.

## Are there API Changes?
No
## Is this a breaking change?
No
## Anything else?
Other comments, collaborators, etc.
Collaborated with @Nikhil-Samant.

This PR Closes Issue #

### CODE REVIEW

**Comment by Willmish** [comment]
Hey @vamsi12a2 can you rebase this on `dev` ?

**Comment by vamsi12a2** [comment]
Hi @Willmish, I rebased this branch on dev.

**Review by vaughanknight — APPROVED** [review]
Looks good.  Much cleaner thanks!

**Review by vaughanknight — APPROVED** [review]
Looks good to me.  Much cleaner thanks.

File: src/CarbonAware.WebApi/src/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-The Carbon Aware SDK provides an API to get the marginal carbon intensity for a given location and time period. The values reported in the Green Software Foundation's specification for marginal carbon intensity (Grams per Kilowatt Hour). In order to use the API, the environment needs to be prepared with a set of configuration parameters. Instructions on setting up the environment could be found here - [GettingStarted.md](../../../GettingStarted.md)
-
 # Carbon Aware REST API
+The Carbon Aware SDK provides an API to get the marginal carbon intensity for a given location and time period. The values reported in the Green Software Foundation's specification for marginal carbon intensity (Grams per Kilowatt Hour). In order to use the API, the environment needs to be prepared with a set of configuration parameters. Instructions on setting up the environment could be found here - https://github.com/microsoft/carbon-aware-sdk/blob/dev/GettingStarted.md
```
Comment (vaughanknight) [inline src/CarbonAware.WebApi/src/README.md:2]: Just noticed that the URL is absolute and only points to dev branch, and is not formatted and is simply a raw link.  Please update to previous format with relative link to document.

**Review by vaughanknight — CHANGES_REQUESTED** [review]
Changes look good however the link to the getting started file is absolute and to dev branch only.  Please use relative links where possible and the document name (and not a raw link) as it heavily impacts on accessibility and readability.

File: src/CarbonAware.WebApi/src/README.md (hunk 1 above)
Comment (vaughanknight) [reply src/CarbonAware.WebApi/src/README.md:2]: I resolved the other conflicts, but just sending this one back for visibilty/process.

Comment (vamsi12a2) [reply src/CarbonAware.WebApi/src/README.md]: Hi @vaughanknight, fixed GettingStarted.md link.

**Comment by danuw** [comment]
LGTM

**Pull request merged**